tests-alloc/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
```

Comment:
This is copying the existing `tests` crate, right?  Could this be done without the duplication by adding a feature flag to `tests`, and then running it in CI with and without that flag?  That would also let you extend the coverage to tests-2015 easily.

Reply:
I think it can't quite be done that way, because `libtest` brings in a dependency on rust std.

I was trying to build a test that shows that we can exercise all of the prost functionality without bringing in std at all.

But the test that I created doesn't actually quite prove that, it isn't guaranteed to fail if std comes in because I didn't figure out a good way to enforce that.

Like I said here (https://github.com/danburkert/prost/pull/215#issuecomment-593708145) If you would prefer to just nuke `tests-alloc` entirely that might be a good idea